packages/circuit-ui/components/Input/Input.tsx

```diff
@@ -108,7 +108,7 @@ export interface InputProps extends Omit<HTMLProps<HTMLInputElement>, 'label'> {
   /**
    * The ref to the html dom element
    */
-  ref?: Ref<HTMLInputElement & HTMLTextAreaElement>;
+  ref?: Ref<HTMLInputElement | HTMLTextAreaElement>;
```

Comment:
This doesn't play well with TypeScript. There are two possible solutions: 

1. Use `Ref<HTMLInputElement & HTMLTextAreaElement>` in your code.
2. Change the Input `ref` to `Ref<any>`. We've done the same for the Text component recently.